regression-tests/25-compile-c64/Makefile

```diff
@@ -0,0 +1,14 @@
+EXAMPLESDIR=../../examples
+TOOLSDIR=../../tools
+
+# build c64 tests. Many examples won't fit in memory or won't compile due to the
+# lack of int64_t in cc65, so only a subset of IPv4 is tested.
+EXAMPLES = irc/c64 \
+telnet-server/c64 \
+webbrowser/c64 \
+webserver/c64 \
+wget/c64
```

Comment:
Ok, I didn't notice the already existent tests. Sorry for the mess, I should have paid more attention there.

hello-world currently compiles on my side when UIP_ACTIVE_OPEN = 1. When it is disabled, Contiki fails to build [on this line](https://github.com/contiki-os/contiki/blob/master/core/net/ipv6/uip6.c#L1858) since [tcp_send_syn is undefined](https://github.com/contiki-os/contiki/blob/master/core/net/ipv6/uip6.c#L1798). Enclosing #if/#endif are missing there, I believe surrounding lines 1857/1858 would solve the problem without breaking anything else, could anyone confirm that? I am adding this change to this pull request.

Linkage fails due to code size (overflows by 4849 bytes). Let me know if you have any suggestions to reduce code size.